test/usecases/implicit_argument/cacur.mod

```diff
@@ -0,0 +1,27 @@
+NEURON {
+	SUFFIX cacur
+	USEION ca WRITE ica
```

Comment:
Why mixed with ions?

Reply:
It's this file here: https://github.com/neuronsimulator/nrn/blob/master/share/examples/nrniv/nmodl/cacur.mod

Reply:
Good to know. We have Model DB CI to check a variety of scientific examples. In the "usecases" we'd like to break down each language construct into the minimum needed to test it. Here `at_time` is likely not related to ions. Please strip everything unrelated. Then simplify such that we can use the MOD file to check one thing at a time.

Do you know why it's called `implicit_argument`? We've (mostly) switch calling the directory by the name of the keyword it's testing, i.e. `at_time`. (If multiple keywords are required in a test we pick one.)

Reply:
>  In the "usecases" we'd like to break down each language construct into the minimum needed to test it.

In this case, I don't think explicit testing is required, what's important is that it compiles since it's an internal NEURON function (so I assume the functionality itself is tested somewhere in the NEURON test suite).

> Do you know why it's called implicit_argument?

Yes, it's because it's a consequence of the `ImplicitArgumentVisitor`:
https://github.com/BlueBrain/nmodl/blob/5ff221d045c92389b4cab5ee159d46770aa12665/src/main.cpp#L536-L538